config/racecar.rb

```diff
@@ -3,13 +3,13 @@
   config.group_id_prefix = 'compliance'
 
   if Settings.kafka.security_protocol
-    config.security_protocol = Settings.kafka.security_protocol.try(:to_sym)
+    config.security_protocol = Settings.kafka.security_protocol.try(:downcase).try(:to_sym)
 
     if config.security_protocol == :sasl_ssl
       config.ssl_ca_location = Settings.kafka.ssl_ca_location
```

Comment:
Can the CA be also set for security protocol `:ssl`?

Reply:
I need a bit more explanation for this :thinking: